browser/css/jsdialogs.css

```diff
@@ -1184,6 +1184,12 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	max-height: 40vh;
 }
 
+#BorderPage.jsdialog #presets, 
```

Comment:
can we have it at the bottom of the file with comment indicating how to find that element in the ui?
similar to `/* Writer - Insert - Page Number - page preview image */`

because later it is hard to know where it was needed, also we try to keep `general widget` rules here, and `exceptions for particular instances` at the end of file separated by comment

Reply:
if `#shadows` is icon view -> good to put it's class there to know it was icon view widget